packages/frontend/styles/theme.ts

```diff
@@ -244,6 +245,21 @@ const theme = createTheme(colorTheme, {
             padding: "0.438rem 1rem",
           },
         },
+        {
+          props: {
+            variant: "small",
+          },
+          style: {
+            marginLeft: "0.5rem",
```

Comment:
marginLeft is very specific, therefore you may want to specify it directly in Typography.

I think in general external margin need to be specified on the go cause they depends on context